language-design.md

```diff
@@ -43,6 +123,12 @@ In what way is an API a language?
 
 **Response**
 
+The easy answer is that it is a mode of communication between two or more things, that provides an 
+abstract way of communicating more complex concepts. Like other languages and programming 
+languages, it has a fixed, unambivalent syntax, is used by computers to interface with other 
+computers, and is documented so that users can learn how to utilize it in their own programs 
+[Bloch, 2006].
+
```

Comment:
If that's the easy answer (which I agree with), do you have a harder answer? (just curious)